php/example_code/lightsail/CreateInstances.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * ABOUT THIS PHP SAMPLE => This sample is part of the SDK for PHP Developer Guide topic at
+ *
+ *
+ */
+// snippet-start:[lightsail.php.create_instance.complete]
+// snippet-start:[lightsail.php.create_instance.import]
+require 'vendor/autoload.php';
+
+use Aws\Exception\AwsException;
+use Aws\Lightsail\LightsailClient;
+
+// snippet-end:[lightsail.php.create_instance.import]
+
+/**
+ * Create an Amazon Lightsail Instance.
+ *
+ * This code expects that you have AWS credentials set up per:
+ * https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials.html
+ */
+
+//Create a Lightsail Client
+// snippet-start:[lightsail.php.create_instance.main] 
+$client = new Aws\Lightsail\LightsailClient([
+    'profile' => 'default',
+    'version' => '2016-11-28',
+    'region' => 'us-east-2'
+]);
+
+$availabilityZone = 'us-east-2a';

```

Comment:
There's no need to define these four as variables, since they're only used once. You can just include the values directly in the `createInstances` call below. Applies to all examples where the variable is only used once.